### PR TITLE
Create powershell.toml

### PR DESCRIPTION
**Pull request by Codemonkey51**
add powershell

not: i didnt know where to put the binary, i can move that if required

### CODE REVIEW

**Comment by Stekovaya** [comment]
`/usr/bin/` is for put binary :)

**Comment by Codemonkey51** [comment]
Ok thx

**Comment by Codemonkey51** [comment]
Ughhh one test failed bc npm wasn't installed

**Comment by kognise** [comment]
(I don't know if I'm allowed to do that but it looks good lmfao)

**Comment by Codemonkey51** [comment]
YAY
now i just need it to be merged

File: languages/powershell.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name = "powershell"
+entrypoint = "main.ps1"
+extensions = [
+  "ps1"
```
Comment (ALANVF) [inline languages/powershell.toml:4]: Powershell has more file extensions than just `.ps1`. Look [here](https://richardspowershellblog.wordpress.com/2017/10/30/powershell-file-extensions/) for a list of the file extensions.

Comment (Codemonkey51) [reply languages/powershell.toml:4]: ok will di

File: languages/powershell.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "psm1",
+  "psd1"
+]
+packages = [
+]
+setup = [
+  "curl -L -o /tmp/powershell.tar.gz https://github.com/PowerShell/PowerShell/releases/download/v7.0.2/powershell-7.0.2-linux-x64.tar.gz",
+  "mkdir /usr/local/pwsh",
+  "tar zxf /tmp/powershell.tar.gz -C /usr/local/pwsh",
+  "ln -s -f /usr/local/pwsh/pwsh /usr/bin/pwsh"
```
Comment (dan-stowell) [inline languages/powershell.toml:14]: Can you also remove `/tmp/powershell.tar.gz`?

Comment (Codemonkey51) [reply languages/powershell.toml:14]: ok one sec

Comment (Codemonkey51) [reply languages/powershell.toml]: fixed

**Comment by Codemonkey51** [comment]
yay

**Comment by Codemonkey51** [comment]
@dan-stowell it passed

**Pull request merged**